third_party/airflow/tox.ini

```
@@ -9,22 +9,24 @@ extras = test
 whitelist_externals =
     bash
     find
-    xargs 
+    xargs
 commands =
     pytest tests/unit
     # This executes the dag files in examples but really only checks for imports and python errors
     bash -c "find examples/ -maxdepth 1 -type f -name *.py | xargs python3"
 
 [testenv:format]
-extras = format 
-commands = 
+extras = format
+commands =
     black --check armada/operators tests/ examples/
+    mypy --exclude '^(.*_pb2.*\.py$|.*_pb2.*\.pyi$|.*typings.py)$' --ignore-missing-imports armada/operators tests/ examples/
```

```suggestion
    mypy --exclude '^(.*_pb2.*\.py$|.*_pb2.*\.pyi$' --ignore-missing-imports armada/operators tests/ examples/
```

We don't have typings for airflow operator.